data/sql/updates/pending_db_world/rev_1556033550219221000.sql

```diff
@@ -0,0 +1,54 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1556033550219221000');
+
+-- Creating a new SmartAI script for [Creature] ENTRY 1998 (name: Webwood Lurker)
+
+-- Table creature_template
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 1998;
+
+-- Table smart_scripts
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 1998);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(1998, 0, 0, 0, 0, 0, 100, 0, 2300, 2300, 17600, 17600, 11, 11918, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Webwood Lurker - In Combat - Cast \'11918\'');
```

Comment:
Change [11918](https://wotlk.evowow.com/?spell=11918) to [744](https://wotlk.evowow.com/?spell=744). That is correct ability for this NPC